elements/urth-core-dataframe/urth-core-query-agg.html

```diff
@@ -0,0 +1,83 @@
+!--
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+-->
+<link rel="import" href="../polymer/polymer.html">
+
+<!--
+This element is used to define an aggregate function to performend on grouped data. It is intended
+as a child element of `<urth-core-query-group`>. The `op` property defines a supported aggregate
+function in for the type of DataFrame. The `col` property is a valid column name for which to perform
+the aggregation. Please refer to the following for details:
+
+- Pandas DataFrame: http://pandas.pydata.org/pandas-docs/version/0.16.0/groupby.html#applying-different-functions-to-dataframe-columns
+- PySpark DataFrame: http://spark.apache.org/docs/latest/api/python/pyspark.sql.html#pyspark.sql.GroupedData.agg
+- Scala Spark DataFrame: https://spark.apache.org/docs/1.4.0/api/java/org/apache/spark/sql/GroupedData.html#agg(org.apache.spark.sql.Column,%20scala.collection.Seq)
+- R DataFrame: https://stat.ethz.ch/R-manual/R-devel/library/stats/html/aggregate.html
+- SparkR DataFrame: https://spark.apache.org/docs/latest/api/R/groupBy.html
+
+Example:
+
+```
+<urth-core-dataframe ref="someDf">
+    <urth-core-query-group by="{{aCol}}">
+        <urth-core-query-agg op="sum" col="someCol"></urth-core-query-agg>
+    </urth-core-query-group>
+<urth-core-dataframe>
+```
+
+@group Urth Core
+@element urth-core-query-agg
+-->
+<dom-module id="urth-core-query-agg">
+    <style>
+        :host {
+            display: none;
+        }
+    </style>
+    <template><content></content></template>
+</dom-module>
+<script>
+
+    (function() {
+        'use strict';
+
+        window.Urth = window.Urth || {};
+
+        window.Urth['urth-core-query-agg'] = Polymer({
+            is: 'urth-core-query-agg',
+
+            properties: {
+                /**
+                 * Name of aggregate operation
+                 */
+                op: {
+                    type: String
+                },
+
+                /**
+                 * Name of column
+                 */
+                col: {
+                    type: String
+                },
+
+                /**
+                 * True if both op and col are set
+                 */
+                valid: {
```

Comment:
sorry, incorrectly suggested `readOnly` for `valid` which is a computed property.